R/auth.R

```diff
@@ -147,7 +147,7 @@ showUsers <- function(appDir=getwd(), appName=NULL, account = NULL,
   accountDetails <- accountInfo(resolveAccount(account, server), server)
 
   if (!isCloudServer(accountDetails$server)) {
-    stop("This method only works for ShinyApps or rstudio.cloud servers.")
+    stop("This method only works for ShinyApps or posit.cloud servers.")
```

Comment:
should we use `shinyapps.io` instead of `ShinyApps`?  we use `shinyapps.io` in other places

Reply:
Good point, these names are incorrect and not consistent with each other. I'll tweak.

Reply:
I'm just going to revert [87f4f63](https://github.com/rstudio/rsconnect/pull/637/commits/87f4f63facef9c3eb24d11e20022d059c7e81194) to keep this PR focused. We can answer the above question elsewhere.